assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/SpreadsheetExporter.java

```diff
@@ -71,7 +73,14 @@ public static SpreadsheetExporter getInstance(Type type, String title, String gr
 
     CsvExporter(String title, String gradeType) {
         gradesBAOS = new ByteArrayOutputStream();
-        gradesBuffer = new CSVWriter(new OutputStreamWriter(gradesBAOS));
+        String defaultSeparator  = ",";
+        //If the decimal separator is a comma
+        if (",".equals(FormattedText.getDecimalSeparator())) {
+        	defaultSeparator = ";";
+        }
+
+        String csvSep = BaseAssignmentService.m_serverConfigurationService.getString("csv.separator", defaultSeparator);
```

Comment:
The above lines are duplicated in both files. Please extract into a common method, getCsvSeparator